apiserver/plane/app/views/analytic/advance.py

```diff
@@ -0,0 +1,425 @@
+from rest_framework.response import Response
+from rest_framework import status
+
+from plane.app.views.base import BaseAPIView
+from plane.app.permissions import ROLE, allow_permission
+from plane.db.models import (
+    WorkspaceMember,
+    Project,
+    Issue,
+    Cycle,
+    Module,
+    IssueView,
+    ProjectPage,
+)
+
+from django.db.models import (
+    Q,
+    Count,
+)
+from plane.utils.build_chart import build_analytics_chart
+from datetime import timedelta
+from django.db.models.functions import TruncDay
+from plane.bgtasks.analytic_plot_export import export_analytics_to_csv_email
+from plane.utils.date_utils import (
+    get_analytics_filters,
+)
+
+
+class AdvanceAnalyticsEndpoint(BaseAPIView):
+    def initialize_workspace(self, slug):
```

Comment:
@NarayanBavisetti Since all the views have the same function, let's create a base view and inherit from it. It'll be easier to maintain.